src/commands/remove-demo.ts

```diff
@@ -1,9 +1,25 @@
 import { GluegunToolbox } from "gluegun"
-import { demo } from "../tools/demo"
+import * as pathlib from "path"
 import type { CommentType } from "../tools/demo"
-import { createGetAllFilePaths } from "../tools/path"
+import { demo } from "../tools/demo"
 import { p, warning } from "../tools/pretty"
 
+const MATCHING_GLOBS = [
+  "!**/.expo",
+  "!**/node_modules",
+  "!**/ios/build",
+  "!**/ios/Pods",
+  "!**/android/build",
+  "!**/android/app/build",
+  "!**/.DS_Store",
+  "!**/.expo/**",
+  "!**/node_modules/**",
+  "!**/ios/build/**",
+  "!**/ios/Pods/**",
+  "!**/android/build/**",
+  "!**/android/app/build/**",
+]
```

Comment:
@joshuayoes these are the glob matchers we're going to target. fs-jetpack uses this: https://github.com/isaacs/node-glob